boardgame-ui/src/utils/api/Game.js

```diff
@@ -74,10 +79,15 @@ export default class Game extends Model {
     if(this.isNew()) {
       return super.save();
     }
-    else {
+    else if (this.id_event != null) {
       let {data} = await axios.put(`/event/${this.id_event}/${this.className}/${this.id}`, this.getPublicProperties());
```

Comment:
I think we should remove this endpoint and always use the same one to update, what do you think? (maybe ask @waliens as well?)

Reply:
Yes, I was thinking the same thing. It was one of the things I wanted you to have a look on in this PR. :)

Reply:
There are different access checks that are applied in both cases so it make sense to keep both endpoints.